members/views.py

```diff
@@ -1,9 +1,27 @@
-from django.contrib.auth import get_user_model
+import bcrypt
+from rest_framework import status
 from rest_framework.generics import CreateAPIView
-from .serializers import SignupSerializer
-from members import serializers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django.contrib.auth.hashers import check_password
+import jwt
+from .models import User
+from .serializers import UserSerializer
+from secret import SECRET
+
 
 # Create your views here.
 class SignupView(CreateAPIView):
-	model = get_user_model()
-	serializer_class = SignupSerializer
+	model = User
+	serializer_class = UserSerializer
+
+
+class SigninView(APIView):
+	def post(self, request, format=None):
+		user = User.objects.get(email=request.data["email"])
+		if not user:
+			return Response(" 존재하지 않는 이메일 ", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
존재하지 않는 이메일이라면 `404 NotFound`나 `401 Unauthorized`가 더 어울릴 것 같습니다.
```suggestion
			return Response(" 존재하지 않는 이메일 ", status=status.HTTP_404_NOT_FOUND)
```

```suggestion
			return Response(" 존재하지 않는 이메일 ", status=status.HTTP_401_UNAUTHORIZED)
```